tasks/section5.yml

```diff
@@ -11,15 +11,15 @@
       - rule_5.1.1
 
 - name: "SCORED | 5.1.1 | PATCH | Ensure cron daemon is enabled"
-  command: /bin/true
+  service: name=crond enabled=yes
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md